code/obj/item/sponge_capsule.dm

```diff
@@ -31,7 +31,7 @@
 					/mob/living/critter/brullbar,
 					/obj/critter/bat/buff,
 					/mob/living/critter/spider/ice,
-					/obj/critter/townguard/passive,
+					/mob/living/critter/townguard,
```

Comment:
Is the change to non-passive intentional?

Reply:
Yeah I changed this one to agressive since the other spawns are.